src/main/kotlin/kotlinx/kover/api/KoverExtension.kt

```diff
@@ -8,6 +8,17 @@ import org.gradle.api.model.*
 import org.gradle.api.provider.*
 
 open class KoverExtension(objects: ObjectFactory) {
+
+    /**
+     * Specifies whether the plugin is applied to the test task and configures it to collect and generate coverage data.
+     */
+    public var isEnabled: Boolean = true
+
+    /**
+     * Specifies the coverage engine to be used to collect execution data.
+     */
+    public val coverageEngine: Property<CoverageEngine> = objects.property(CoverageEngine::class.java)
```

Comment:
haven't you thought about using some sort of sealed class to specify engine+version?
smth like:
```kotlin
sealed class CoverageEngine {
  class IntelliJ(val version: String): CoverageEngine()
  class JaCoCo(val version: String): JaCoCo()
}
```

Reply:
Thanks, we will consider this option for the next release.